### ISSUE TITLE
@mantine/core Text component color is not inherited

### ISSUE THREAD

**Issue by labithiotis** [mantinedev/mantine#402]
It seems like the `Text` component does not inherit color value from parents by default and only if we set `sx={{ color: "inherit" }}>` does it work 

Is that intended? 

Codesanbox example: https://codesandbox.io/s/mantine-bug-h26lc

Would it be terrible to alter `Text` source code to default `color` to `inherited`? https://github.com/mantinedev/mantine/blob/master/src/mantine-core/src/components/Text/Text.tsx#L61

**Comment by rtivital** [comment]
If you want Text to inherit color, you can use the `sx` as you described above. In most cases where you would want to inherit color you can replace Text with just a div element or Box component.

**Comment by labithiotis** [comment]
The only slight issue with using a div/span element is the font family isn't used either, I would need to also apply a CSS style to set font family on body.
https://codesandbox.io/s/mantine-bug-h26lc?file=/src/App.tsx 
![image](https://user-images.githubusercontent.com/784306/141099752-e0859941-b511-4142-83cd-40ccc0b562a9.png)

**Comment by labithiotis** [comment]
Oh nice, I missed that component in docs. That works except for one thing, I have set some custom font sizes which don't seem to be applied by GlobalStyles source 

```jsx
  return (
    <MantineProvider theme={{ fontSizes: { xs: 10, sm: 11, md: 12, lg: 16, xl: 18 } }}>
      <GlobalStyles />
      <div style={{ fontSize: 12 }}>
        <App />
      </div>
    </MantineProvider>
  );
  ```


Would it be ok to amend the GlobalStyles body to use `md` font size from theme?
```ts
body {
  fontSize: theme.fontSizes.md,
}
```
https://github.com/mantinedev/mantine/blob/master/src/mantine-styles/src/theme/GlobalStyles.tsx

**Comment by rtivital** [comment]
Yeah, sounds fine

**Comment by labithiotis** [comment]
Added a PR that updates GlobalStyles: https://github.com/mantinedev/mantine/pull/403

**Comment by rtivital** [comment]
Feature was released in 3.2
